noir-projects/aztec-nr/aztec/src/macros/aztec.nr

```diff
@@ -19,8 +24,9 @@ pub comptime fn aztec(m: Module) -> Quoted {
     // with `static_assert(false, ...)` to prevent them from being called directly from within the contract.
     let functions = process_functions(m);
 
+    // We generate structs and their implementations necessary for convenient functions calls.
     let interface = generate_contract_interface(m);
-    let self_call_structs = generate_self_call_structs(m);
+    let self_call_structs = generate_external_function_self_calls_structs(m);
```

Comment:
Somewhat unrelated but felt like the original function name was no longer descriptive enough.